shared/src/data/default-state/patient-templates.ts

```diff
@@ -12,173 +13,1210 @@ const defaultMaleImage: ImageProperties = {
+            minimumHealth: healthPointsDefaults.greenAverage,
+        },
+    ]
+);
+
+const recoverWithRSState = PatientHealthState.create(
+    FunctionParameters.create(
+        0,
+        calculateHealthChange(
+            healthPointsDefaults.yellowAverage,
+            healthPointsDefaults.yellowMax,
+            1
+        ),
+        calculateHealthChange(
+            healthPointsDefaults.yellowAverage,
+            healthPointsDefaults.yellowMax,
+            1
+        ),
+        calculateHealthChange(
+            healthPointsDefaults.yellowAverage,
+            healthPointsDefaults.yellowMax,
+            1
+        )
+    ),
+    [
+        {
+            matchingHealthStateId: noChangesState.id,
+            minimumHealth: healthPointsDefaults.greenAverage,
+        },
+    ]
+);
+
+const yellowUntilPhase4State = PatientHealthState.create(
+    FunctionParameters.create(0, 0, 0, 0),
+    [
+        {
+            matchingHealthStateId: recoverWithRSState.id,
+            earliestTime: 4 * phaseTime,
+        },
+    ]
+);
+
+const yellowStartPhase4RSDecisionState = PatientHealthState.create(
+    FunctionParameters.create(
+        calculateHealthChange(
+            healthPointsDefaults.yellowAverage,
+            healthPointsDefaults.redAverage,
+            1
+        ),
+        -calculateHealthChange(
+            healthPointsDefaults.yellowAverage,
+            healthPointsDefaults.redAverage,
+            1
+        ),
+        -calculateHealthChange(
+            healthPointsDefaults.yellowAverage,
+            healthPointsDefaults.redAverage,
+            1
+        ),
+        -calculateHealthChange(
+            healthPointsDefaults.yellowAverage,
+            healthPointsDefaults.redAverage,
+            1
+        )
+    ),
+    [
+        {
+            matchingHealthStateId: noChangesState.id,
+            maximumHealth: healthPointsDefaults.redMax,
+        },
+    ]
+);
+
+const yellowUntilRedPhase4State = PatientHealthState.create(
+    FunctionParameters.create(0, 0, 0, 0),
+    [
+        {
+            matchingHealthStateId: yellowStartPhase4RSDecisionState.id,
+            earliestTime: 4 * phaseTime,
+        },
+    ]
+);
+
+const yellowStartPhase9RADecisionState = PatientHealthState.create(
+    FunctionParameters.create(
+        calculateHealthChange(
+            healthPointsDefaults.yellowAverage,
+            healthPointsDefaults.blackMax,
+            1
+        ),
+        -calculateHealthChange(
+            healthPointsDefaults.yellowAverage,
+            healthPointsDefaults.blackMax,
+            1
+        ),
+        -calculateHealthChange(
+            healthPointsDefaults.yellowAverage,
+            healthPointsDefaults.blackMax,
+            1
+        ),
+        0
+    ),
+    [
+        {
+            matchingHealthStateId: noChangesState.id,
+            minimumHealth: healthPointsDefaults.yellowAverage,
+        },
+        {
+            matchingHealthStateId: noChangesState.id,
```

Comment:
Just as a general note: This phase change has no effect as dead patients are not considered for treatment and therefore keep their health at `0`. But this approach works, too.